core/conn/jdbcT4/src/main/java/org/trafodion/jdbc/t4/TrafT4ResultSet.java

```diff
@@ -1372,7 +1372,7 @@ public long getLong(int columnIndex) throws SQLException {
 						}
 					} else {
 						throw TrafT4Messages.createSQLException(connection_.props_, connection_.getLocale(),
-								"numeric_out_of_range", null);
+								"numeric_out_of_range", bd.longValue());
```

Comment:
Will the longValue convert the bd into  a proper long value? Should we display the original bd value instead?